src/main/resources/FormSectionProfile.json

```diff
@@ -5,6 +5,7 @@
       {
         "fieldName": "Order",
         "metadataPropertyName": "order",
+        "description": "Numerical order for sections in this form",
```

Comment:
Ditto the comment about the order on the FormItemProfile, I think this should match the text of that.

Reply:
Done